### PR TITLE
upgrade packages

### PR DESCRIPTION
**Pull request by colinwitkamp**
Resolved:
 - Upgrade all packages to the latest.
 - Set up ESLint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,23 +9,22 @@
     "test-integration": "cross-env NODE_ENV=test ./node_modules/.bin/mocha test/integration/*.test.js"
   },
   "dependencies": {
-    "bluebird": "^3.5.1",
-    "body-parser": "^1.18.2",
-    "cookie-parser": "^1.4.3",
-    "debug": "^3.1.0",
-    "express": "^4.16.2",
-    "morgan": "^1.7.0",
-    "pg": "^6.1.0",
```
Comment (papb) [inline package.json:18]: Why did you remove `pg`?

**Comment by colinwitkamp** [comment]
I think we don't need `pg` as it is using `sqlite` for db. To make it
minimalistic, I removed it.

**Comment by papb** [comment]
@colinwitkamp Thanks, that makes sense. I will not merge yet just to make some other tests, I plan to improve this package as a whole in the near future.

**Comment by sushantdhiman** [comment]
@papb Nice, Actually it will make sense to convert this repo to a multi-repo with various examples. We can start with `examples/express` as a start and add other examples like `examples/express-with-typescript` etc

**Comment by colinwitkamp** [comment]
okay, no problem,

**Comment by papb** [comment]
Thank you!! :tada:
